### ISSUE TITLE
[Bug]: Items/ Actions in Sharing Dialog Window not accessible without Mouse

### ISSUE THREAD

**Issue by tigexplorer** [nextcloud/desktop#5836]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

The share dialog is not accessible without a mouse.
...

### Steps to reproduce

1. Right click on the file
2. Click on Share Options
3. Only the TabWidget? with **Activity** and the **Share** is available with the keyboard (left/ right arrow). 
4. It's not visible, what item has the focus.
5. If one press `Enter` sometimes the internal link is copied.
6. I don't find a way, to perform generating a new share link, not with space or tab.
7. It seems there is no tab index at all.
...

### Expected behavior

When clicking on the share dialog, the selected item/ action may have a visible focus AND can access with Tab/ Space or usual keybord keys.
Also it would be good, when shortcuts and a obvious legend therefore can implemented. 
...

### Which files are affected by this bug

all

### Operating system

Linux

### Which version of the operating system you are running.

ubuntu 22 LTS

### Package

Distro package manager

### Nextcloud Server version

25.0.7

### Nextcloud Desktop Client version

3.9.0-20230616.170352.c6c6c61f5-1.0~jammy1

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [ ] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other
